### PR TITLE
💄New Design - Upcoming events 

### LINKED ISSUE #1148: 💸 Tech Debt - Improve "Upcoming Events" section
cc @bradystroud @BrookJeynes @jaydenalchin @adamcogan @amankumarrr 

**Description**  

"Upcoming Events" section is currently too cluttered...

**Proposed solution**  

- [ ]  Action the changes in this video (same as below): https://www.loom.com/share/4234abbbb4454675a83e87d4d4772e04  

1. Add padding to article (using class `p-4`)
2. Replace overall text padding (`px-6`) with a margin left (`ml-4`)
3. Move the countdown tag "DAYS TO GO" to come 1st
    - change margin from left to right (from `ml-2` to `mr-1`)

4. Reduce "time" text size ( `text-xs`)  - This is because of https://github.com/SSWConsulting/SSW.Website/issues/1147

---

Screenshot 2023-07-31 at 12 42 04 PM

**Figure: Current "Upcoming Events" section** 

Screenshot 2023-07-31 at 1 03 54 PM

**Figure: Proposed "Upcoming Events" section after changes**

### PR DESCRIPTION
**Pull request by amankumarrr**
Fixes #1148 

Affected routes:

- `/` 

Add done video, screenshots
image

**Figure: newly designed upcoming event component v1**

image

**Figure: newly designed upcoming event component v2**

image

**Figure: newly designed upcoming event component v2 in mobile view**

### CODE REVIEW

**Comment by jaydenalchin** [comment]
Hi @amankumarrr CC @tiagov8 

1. We shouldn't have a scrollbar here. I would just show the 5 upcoming events as demonstrated above.
2. To compensate, we should add more emphasis to the "More Events" button
3. Add a verb to the button i.e. "**See** more events" to follow best practice
4. The card border colour is too light & low contrast against white, it should be #CCCCCC
5. Increase the gap between these two sections. The grey events bg was doing a lot of work separating the upcoming events section from the text-heavy section to the left.
6. These headings should also be h2 not h1, and match the h2's "Contact Us" / "tv.ssw.com" in the section below this. 

Here's an example of some of my feedback actioned in my browser: 

image

**Figure: Section gap (5rem), red button, updated button text**

**Review by bradystroud — APPROVED** [review]
LGTM - love the new design

File: pages/[filename].tsx [hunk 1]
```diff
@@ -47,8 +47,8 @@ export default function HomePage(
           <></>
         )}
         <Blocks prefix="PageBeforeBody" blocks={data.page.beforeBody} />
-        <Container className="flex-1 pt-4">
-          <div className="gap-4 md:grid md:grid-cols-5">
+        <Container className={"flex-1 pt-4"}>
```
Comment (jaydenalchin) [inline pages/[filename].tsx]:
```suggestion
        <Container className={"flex-1"}>
```

**Review by jaydenalchin — CHANGES_REQUESTED** [review]

**Comment by jaydenalchin** [comment]
@amankumarrr @bradystroud

1. The cards themselves should be clickable, not just the title text
2. The cards need hover styling. This is still TBD for the proposed rules cards - I would suggest just adding border-colour #333333 on hover and we can reevaluate later if needed for consistency.

**Comment by amankumarrr** [comment]
> @amankumarrr @bradystroud
> 
> 1. The cards themselves should be clickable, not just the title text
> 2. The cards need hover styling. This is still TBD for the proposed rules cards - I would suggest just adding border-colour #333333 on hover and we can reevaluate later if needed for consistency.

@jaydenalchin  Done

**Review by jaydenalchin — APPROVED** [review]
LGTM 🚀

**Pull request merged**